bcipy/parameters/devices.json

```diff
@@ -2,58 +2,91 @@
+        "channels": [
+            { "name": "P3", "label": "P3", "units": "microvolts", "type": "EEG" },
+            { "name": "C3", "label": "C3", "units": "microvolts", "type": "EEG" },
+            { "name": "F3", "label": "F3", "units": "microvolts", "type": "EEG" },
+            { "name": "Fz", "label": "Fz", "units": "microvolts", "type": "EEG" },
+            { "name": "F4", "label": "F4", "units": "microvolts", "type": "EEG" },
+            { "name": "C4", "label": "C4", "units": "microvolts", "type": "EEG" },
+            { "name": "P4", "label": "P4", "units": "microvolts", "type": "EEG" },
+            { "name": "Cz", "label": "Cz", "units": "microvolts", "type": "EEG" },
+            { "name": "Pz", "label": "Pz", "units": "microvolts", "type": "EEG" },
+            { "name": "Fp1", "label": "Fp1", "units": "microvolts", "type": "EEG" },
+            { "name": "Fp2", "label": "Fp2", "units": "microvolts", "type": "EEG" },
+            { "name": "T3", "label": "T3", "units": "microvolts", "type": "EEG" },
+            { "name": "T5", "label": "T5", "units": "microvolts", "type": "EEG" },
+            { "name": "O1", "label": "O1", "units": "microvolts", "type": "EEG" },
+            { "name": "O2", "label": "O2", "units": "microvolts", "type": "EEG" },
+            { "name": "X3", "label": "X3", "units": "microvolts", "type": "EEG" },
+            { "name": "X2", "label": "X2", "units": "microvolts", "type": "EEG" },
+            { "name": "F7", "label": "F7", "units": "microvolts", "type": "EEG" },
+            { "name": "F8", "label": "F8", "units": "microvolts", "type": "EEG" },
+            { "name": "X1", "label": "X1", "units": "microvolts", "type": "EEG" },
+            { "name": "A2", "label": "A2", "units": "microvolts", "type": "EEG" },
+            { "name": "T6", "label": "T6", "units": "microvolts", "type": "EEG" },
+            { "name": "T4", "label": "T4", "units": "microvolts", "type": "EEG" },
+            { "name": "TRG", "label": "TRG", "units": "microvolts", "type": "EEG "}
+        ],
         "sample_rate": 300.0,
         "description": "Wearable Sensing DSI-24",
         "excluded_from_analysis": ["TRG", "X1", "X2", "X3", "A2"]
     },
     {
         "name": "DSI-VR300",
         "content_type": "EEG",
-        "channels": [ "P4", "Fz", "Pz", "F7", "PO8", "PO7", "Oz", "TRG" ],
+        "channels": [
+            { "name": "P4", "label": "P4", "units": "microvolts", "type": "EEG" },
+            { "name": "Fz", "label": "Fz", "units": "microvolts", "type": "EEG" },
+            { "name": "Pz", "label": "Pz", "units": "microvolts", "type": "EEG" },
+            { "name": "F7", "label": "F7", "units": "microvolts", "type": "EEG" },
+            { "name": "PO8", "label": "PO8", "units": "microvolts", "type": "EEG" },
+            { "name": "PO7", "label": "PO7", "units": "microvolts", "type": "EEG" },
+            { "name": "Oz", "label": "Oz", "units": "microvolts", "type": "EEG" },
```

Comment:
Is that how LSL outputs it? "microvolts"? I'm okay with either but usually, I only see the abbreviation.

Reply:
Yeah, the units that come across depend on what the adapter application sends. In this case, it's dsi2lsl that's writing these metadata values. See:

https://github.com/BciPy/dsi2lsl/blob/a94876c1977174d6257997f7f25d4138f11d7891/dsi2lsl.c#L343